<launch>
  	<node 	pkg="obstacle_estimator"
    		type="obstacle_prediction_node"
    		name="obstacle_prediction_node_1"
    		respawn="false"
    		launch-prefix=""
    		required="true">
			<param name="delta_t" value="0.05" />
			<param name="horizon_N" value="30" />
			
			<remap from="/obstacle/pose" to="/Target1/pose"/>
			<remap from="/obstacle/state_prediction" to="/Target1/state_prediction"/>
  	</node>

	<node 	pkg="obstacle_estimator"
			 type="obstacle_prediction_node"
			 name="obstacle_prediction_node_2"
			 respawn="false"
			 launch-prefix=""
			 required="true">
			 <param name="delta_t" value="0.05" />
			 <param name="horizon_N" value="30" />

			 <remap from="/obstacle/pose" to="/Target2/pose"/>
			 <remap from="/obstacle/state_prediction" to="/Target2/state_prediction"/>
	</node>

</launch>
